src/components/DataTable.vue

```diff
@@ -33,16 +33,25 @@
 </template>
 
 <script>
+import MoneyComponent from '@/components/MoneyComponent'
+
 export default {
     name: "DataTable",
 
+    components: {
+      MoneyComponent
+    },
+
     props: {
       bikes: Array
     },
 
     data() {
       return{
         search: '',
+
+        styling: 'text-xs-right',
```

Comment:
Why the extra styling property?

Reply:
removed it, as it was just not that useful for reducing redundency